omod/src/main/java/org/openmrs/module/laboratoryapp/fragment/controller/ResultFragmentController.java

```diff
@@ -73,9 +72,53 @@ public SimpleObject saveResult(
 		test.setEncounter(encounter);
 		test = ls.saveLaboratoryTest(test);
 		ls.completeTest(test);
-		
+
+		this.sendPatientToQueue(encounter);
+
 		return SimpleObject.create("status", "success", "message", "Saved!");
 	}
+
+	private void sendPatientToQueue(Encounter encounter)
+	{
+		//send patient back to the OPD Queue
+		Patient patient = encounter.getPatient();
+		PatientQueueService queueService = Context.getService(PatientQueueService.class);
+		Concept referralConcept = Context.getConceptService().getConcept(2548);
```

Comment:
Couldnt find the concept anywhere in the lab constants


 public static final String MODULE_ID = "laboratory";
    public static final String TEST_STATUS_ACCEPTED = "accepted";
    public static final String TEST_STATUS_COMPLETED = "completed";
    public static final String UNACCEPT_TEST_RETURN_STATUS_SUCCESS = "success";
    public static final String UNACCEPT_TEST_RETURN_STATUS_NOT_FOUND = "not found";
    public static final String UNACCEPT_TEST_RETURN_STATUS_PRINTED = "printed";
    public static final String RESCHEDULE_TEST_RETURN_STATUS_SUCCESS = "success";
    public static final String RESCHEDULE_TEST_RETURN_STATUS_PRINTED = "printed";
    public static final String RESCHEDULE_TEST_RETURN_STATUS_ENTERED = "entered";
    public static final String COMPLETE_TEST_RETURN_STATUS_SUCCESS = "success";
    public static final String COMPLETE_TEST_RETURN_STATUS_NOT_ACCEPTED = "not accepted";
    public static final String PROPERTY_PRINTWORKLIST_PRINTALLTEST = "laboratory.printWorkList.printAllTest";
    public static final String PROPERTY_PRINTWORKLIST_TESTOPTION_DISPLAY = "laboratory.printWorkList.testOption.display";
    public static final String PROPERTY_PRINTWORKLIST_PATIENTSEARCHBOX_DISPLAY = "laboratory.printWorkList.patientSearchBox.display";
    public static final String PROPERTY_RADIOLOGY_ENCOUNTER = "laboratory.radiologyEncounterType";
    public static final String PROPERTY_MAINTAINCODE = "laboratory.maintainCode";
    public static final String SESSION_TEST_TREE_MAP = "laboratory.testTreeMap";
    public static final Integer PAGESIZE = Integer.valueOf(20);
    public static final Integer ACCEPT_TEST_RETURN_ERROR_EXISTING_SAMPLEID = Integer.valueOf(-2);
    public static final Integer ACCEPT_TEST_RETURN_ERROR_EXISTING_TEST = Integer.valueOf(-1);